nucleus/admin/util/src/main/java/com/sun/enterprise/admin/util/JarFileUtils.java

```diff
@@ -0,0 +1,63 @@
+package com.sun.enterprise.admin.util;
```

Comment:
This file needs the Payara copyright header

Reply:
Sure. I'll fix this.